harmonizome/utility_functions.py

```diff
@@ -343,8 +341,8 @@ def load_data(filename):
             return df
 
 
-def archive(path):
-    with zipfile.ZipFile('output_archive.zip', 'w', zipfile.ZIP_DEFLATED) as zipf:
+def archive(path, output_name=None):
+    with zipfile.ZipFile(output_name+'_output_archive.zip', 'w', zipfile.ZIP_DEFLATED) as zipf:
```

Comment:
This is a breaking change since output_name was not defined before. Please try to modify this so anything which doesn't know about this new parameter still functions as before.